Client/Frontend/Strings.swift

```diff
@@ -46,3 +46,16 @@ extension Strings {
     public static let LoginsHelperDontSaveButtonTitle = NSLocalizedString("LoginsHelper.DontSave.Button", value: "Don’t Save", comment: "Button to not save the user's password")
     public static let LoginsHelperUpdateButtonTitle = NSLocalizedString("LoginsHelper.Update.Button", value: "Update", comment: "Button to update the user's password")
 }
+
+//Hotkey Titles
+extension Strings {
+    public static let ReloadPageTitle = NSLocalizedString("Hotkeys.Reload.DiscoveryTitle", value: "Reload Page", comment: "Label to display in the Discoverability overlay for keyboard shortcuts")
+    public static let BackTitle = NSLocalizedString("Hotkeys.Back.DiscoveryTitle", value: "Back", comment: "Label to display in the Discoverability overlay for keyboard shortcuts")
+    public static let FindTitle = NSLocalizedString("Hotkeys.Find.DiscoveryTitle", value: "Find", comment: "Label to display in the Discoverability overlay for keyboard shortcuts")
+    public static let OpenLocationTitle = NSLocalizedString("Hotkeys.OpenLocation.DiscoveryTitle", value: "Open Location", comment: "Label to display in the Discoverability overlay for keyboard shortcuts")
```

Comment:
I'm not sure if `Open Location` is the right wording here. I'd suggest something more like `Search or Enter Address` which is the placeholder for the Location Text Field.